packages/playground/src/dashboard/dao_view.vue

```diff
@@ -373,7 +373,14 @@ async function castVote() {
       });
       openVDialog.value = false;
     } catch (err) {
-      createToast(`Vote Failed!`, {
+      let errMsg = `Vote Failed!`;
+
+      if (err instanceof TFChainError && err.keyError == "DuplicateVote") {
+        errMsg = `Failed to vote; Already voted`;
+      }
+      console.error(errMsg, err);
```

Comment:
Possible to be another error comes from the chain or another block, let's display an `error message/default error message` in the else block

Reply:
console.error will print on all cases whatever it is a `tfchainError` or not; the check `if (err instanceof TFChainError && err.keyError == "DuplicateVote")` is just for overwrite the errMsg with a specific one